### ISSUE TITLE
Use label mode 'categorical' instead of 'binary' in image_dataset_from_directory

### ISSUE THREAD

**Issue by parmigggiana** [FabioBoccia/Progetto_ESM#2]

**Comment by parmigggiana** [comment]
Se ho capito bene la [documentazione](https://www.tensorflow.org/api_docs/python/tf/keras/utils/image_dataset_from_directory), dovrebbe permettere di usare 2 categorie invece di una sola nell' ultimo layer e quindi usare softmax avendo una probabilita' per entrambe le classi invece che una sola da 0 a 1 (la threshold quale sarebbe?)

**Comment by simone-dotolo** [comment]
Da quanto ho capito la label passa da (batch_size, 0 o 1) a (batch_size, [1,0] o [0,1]) nel nostro caso, quindi del tipo one-hot encoder. Probabilmente se funziona con softmax semplicemente in output da' le due probabilità senza bisogno di una soglia.
Comunque si può provare
